lib/request_handler/fieldsets_parser.rb

```diff
@@ -29,13 +29,18 @@ def run
     private
 
     def parse_options(type, values)
+      raise ExternalArgumentError, fieldsets: "invalid type: <#{type}>" if allowed[type] == false
```

Comment:
one option to acchieve this could be to simply remove all entries from allowed in the initializer with the value false